backend/Controllers/MetadataController.cs

```diff
@@ -13,23 +14,37 @@
 
 using System;
 using System.Net;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
 
 namespace OpenData.Controllers
 {
+    [Authorize]
 	[Route("/api/[controller]")]
 	public class MetadataController : Controller
 	{
 		private readonly IMetadataService _metadataService;
 		private readonly IMapper _mapper;
 		private readonly IUnitOfWork _unitOfWork;
+		private readonly IHttpContextAccessor httpContextRetriever;
+		private readonly IUserService userService;
 
-		public MetadataController(IMetadataService metadataService, IMapper mapper, IUnitOfWork unitOfWork) 
+		public MetadataController(
+            IMetadataService metadataService,
+            IMapper mapper,
+            IUnitOfWork unitOfWork,
+			IHttpContextAccessor httpContextRetriever,
```

Comment:
What's up with the indentions

Reply:
Seems to be related to spaces vs. tabs. Not possible to reproduce locally.